admin/api_alerts.go

```diff
@@ -153,20 +153,20 @@ AcknowledgeAlert Acknowledge One Alert from One Project
 
 Confirms receipt of one existing alert. This alert applies to any component in one project. Acknowledging an alert prevents successive notifications. You receive an alert when a monitored component meets or exceeds a value you set until you acknowledge the alert. To use this resource, the requesting API Key must have the Organization Owner or Project Owner role.
 
-This resource remains under revision and may change.
+This resource remains under revision and may change. Deprecated versions: v2-{2023-01-01}
 
 	@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 	@param groupId Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.  **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
 	@param alertId Unique 24-hexadecimal digit string that identifies the alert. Use the [/alerts](#tag/Alerts/operation/listAlerts) endpoint to retrieve all alerts to which the authenticated user has access.
 	@return AcknowledgeAlertApiRequest
 */
-func (a *AlertsApiService) AcknowledgeAlert(ctx context.Context, groupId string, alertId string, alertViewForNdsGroup *AlertViewForNdsGroup) AcknowledgeAlertApiRequest {
+func (a *AlertsApiService) AcknowledgeAlert(ctx context.Context, groupId string, alertId string, acknowledgeAlert *AcknowledgeAlert) AcknowledgeAlertApiRequest {
```

Comment:
interesting looks like the param was still missing for this specific one?

Reply:
Yes. made fix for master. Looks like new version of openapi has regression

https://github.com/mongodb/atlas-sdk-go/pull/338/files#r1630674739

Reply:
Creating jira to investigate